pkgs/applications/display-managers/sddm/default.nix

```diff
@@ -19,6 +19,10 @@ in mkDerivation {
 
   patches = [
     ./sddm-ignore-config-mtime.patch
+    (fetchpatch {
```

Comment:
please comment why this patch is added, and also link to the PR.

Reply:
Added.